### ISSUE TITLE
Healthcheck for the compute nodes

### ISSUE THREAD

**Issue by wimpers** [openvstorage/openvstorage-health-check#185]
Currently the health check can only run on nodes with the framework. In the ticket https://github.com/openvstorage/openvstorage-health-check/issues/181 we already raised the need to run it on ASD nodes. We should also be able to run the health check on CPU nodes and check the relevant items there (f.e. permission settings, edge client)

**Comment by wimpers** [comment]
Ideally we create single health check which executed checks based upon the installed packages as otherwise we will end up with 3 health checks.

**Comment by matthiasdeblock** [comment]
Indeed, the checks on a master node are different then the checks on a compute node, an extra node or a storage node.

**Comment by matthiasdeblock** [comment]
We faced an issue yesterday that the logfiles on a compute node did not have the correct ownership. A separate health check will detect these issues.

**Comment by kinvaris** [comment]
Why would we need a healthcheck for the compute nodes? The only thing that the compute node contains of OVS is the package libovsvolumedriver and or blktap.

**Comment by matthiasdeblock** [comment]
Checking the correct ownership isn't something for the monitoring (you just can't). Also, on the computenodes we need to make sure the /var/log/syslog kern.log ... must have the correct ownership.

**Comment by khenderick** [comment]
Personally I don't see why we would need a health-check for a compute node. These are the customer's machine, so I'd think it's their own responsibility to make sure they are configured as they should be. If that makes sense, why not extend the health-check so we can make sure the firewalls are setup correctly, or that the soap dispenser in the toilets is sufficiently filled.

**Comment by JeffreyDevloo** [comment]
@wimpers 
Any input on this? I personally agree @khenderick

**Comment by wimpers** [comment]
I do think we need a health check for the compute nodes:
* versions of QEMU (oru version?), EDGE
* check for permissions
* ...

Maybe testing a connection to the voldv? OPS will have more requirements.

**Comment by wimpers** [comment]
In case QEMU is installed we can check the health of a vdisk through qemu info. If there are errors with the vdisk it will or timeout or report IO errors.

Add the option to verify the health of a single vdisk so you can test a subset of vdisks (f.e. random 5% of the vdisks).
